app/components/elements/forms/field_component.rb

```diff
@@ -4,18 +4,20 @@ module Elements
   module Forms
     # Base component for all form fields.
     class FieldComponent < ApplicationComponent
-      def initialize(form:, field_name:, required: false, hidden_label: false, label: nil, help_text: nil, data: {}) # rubocop:disable Metrics/ParameterLists
+      def initialize(form:, field_name:, required: false, hidden_label: false, label: nil, help_text: nil, # rubocop:disable Metrics/ParameterLists
+                     data: {}, **extra_args)
         @form = form
         @field_name = field_name
         @required = required
         @hidden_label = hidden_label
         @label = label
         @help_text = help_text
         @data = data
+        @extra_args = extra_args
         super()
       end
 
-      attr_reader :form, :field_name, :required, :help_text, :hidden_label, :label, :data
+      attr_reader :form, :field_name, :required, :help_text, :hidden_label, :label, :data, :extra_args
```

Comment:
Can you give examples of what's being slurped into `extra_args` in this PR?

Reply:
placeholder

I could be convinced to just add this as an argument.

Reply:
Oh, yeah, that strikes me as a reasonable named arg for a form field component.

Reply:
Named it.